technical-reports/format/design-token.md

```diff
@@ -107,17 +109,35 @@ The **description** property must be a plain JSON string, for example:
 
 ## Type
 
-Declares the type of the token. [See “Types”](#types) for more information.
+Declares the type of the token. [See supported "Types"](#types) for more information.
 
-<div class="issue" data-number="63" title="Token types optional or required">
-  Are token types optional or required?
-</div>
+The `type` property can be set on different levels:
+- at the file level
+- at the group level
+- at the token level
+
+The `type` property must be a plain JSON string.
```

Comment:
@ChucKN0risK we discussed this in our format group meeting and decided we should remove the `- at the file level` bullet, since we haven't discussed how multiple files would work.

The root of the file is explicitly a group.

Reply:
Thanks @c1rrus and @kevinmpowell!

I've updated the PR to:
- include your suggestion @c1rrus 
- remove `- at the file level bullet`